ci/Dockerfile.std

```diff
@@ -1,4 +1,5 @@
 # Generated by: gromit policy
+# Generated on: Wed Apr  3 06:48:24 UTC 2024
```

Comment:
This looks like it is adding the timestamp line, we've ditched the timestamp altogether, right?